xml/System.Text.Json/JsonElement.xml

```diff
@@ -313,9 +349,13 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Gets the current JSON number as a <see cref="T:System.Single" />.</summary>
+        <returns>The current JSON number as a <see cref="T:System.Single" />.</returns>
+        <remarks>This method does not parse the contents of a JSON string value.
```

Comment:
This should be inside a \<format> element and use xrefs as the earlier example did.

Reply:
```suggestion
        <remarks><format type="text/markdown"><![CDATA[
This method does not parse the contents of a JSON string value.
```